homeassistant/components/ombi/sensor.py

```diff
@@ -0,0 +1,125 @@
+"""Support for Ombi."""
+from datetime import timedelta
+import logging
+
+import pyombi
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_API_KEY,
+    CONF_HOST,
+    CONF_MONITORED_CONDITIONS,
+    CONF_PORT,
+    CONF_SCAN_INTERVAL,
+    CONF_SSL,
+)
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_URLBASE = "urlbase"
+
+DEFAULT_PORT = 5000
+DEFAULT_SCAN_INTERVAL = timedelta(seconds=60)
+DEFAULT_SSL = False
+DEFAULT_URLBASE = ""
+
+SENSOR_TYPES = {
+    "movies": {"type": "Movie requests", "icon": "mdi:movie"},
+    "tv": {"type": "TV show requests", "icon": "mdi:television-classic"},
+    "pending": {"type": "Pending requests", "icon": "mdi:clock-alert-outline"},
+    "approved": {"type": "Approved requests", "icon": "mdi:check"},
+    "available": {"type": "Available requests", "icon": "mdi:download"},
+}
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_API_KEY): cv.string,
+        vol.Required(CONF_HOST): cv.string,
+        vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+        vol.Optional(CONF_URLBASE, default=DEFAULT_URLBASE): cv.string,
+        vol.Optional(CONF_SSL, default=DEFAULT_SSL): cv.boolean,
+        vol.Optional(CONF_SCAN_INTERVAL, default=DEFAULT_SCAN_INTERVAL): cv.time_period,
+        vol.Optional(CONF_MONITORED_CONDITIONS, default=list(SENSOR_TYPES)): vol.All(
```

Comment:
We don't allow monitored conditions in config anymore.

https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md

Please create all sensor types by default.

Reply:
Since the sensors make individual API calls, it would seem reasonable to allow the users to limit the number of requests by omitting certain sensors from monitored_conditions. Should I change it anyway? Ombi is a self-hosted service, so the number of API calls are usually not an issue.

Reply:
If it's usually not an issue, I suggest we remove the option.

Reply:
Alright, fixed!